src/java.base/share/classes/sun/security/util/SignatureFileVerifier.java

```diff
@@ -848,15 +848,15 @@ private static int initializeMaxSigFileSize() {
          * in a JAR file.
          */
         Integer tmp = GetIntegerAction.privilegedGetProperty(
-                "jdk.jar.maxSignatureFileSize", 8000000);
+                "jdk.jar.maxSignatureFileSize", 16000000);
         if (tmp < 0 || tmp > MAX_ARRAY_SIZE) {
             if (debug != null) {
-                debug.println("Default signature file size 8000000 bytes " +
+                debug.println("Default signature file size 16000000 bytes " +
                         "is used as the specified size for the " +
                         "jdk.jar.maxSignatureFileSize system property " +
                         "is out of range: " + tmp);
```

Comment:
The wording of the message is a bit hard to understand. I suggest rewording slightly as: 
```
                debug.println("The default signature file size of 16000000 bytes " +
                        "will be used as the specified size for the " +
                        "jdk.jar.maxSignatureFileSize system property " +
                        "since the value is out of range: " + tmp);
```

Reply:
Reworded the debug message.